framework/ramcache.py

```diff
@@ -49,7 +49,10 @@
 
 import logging
 import time as time_module
-from google.appengine.ext import ndb
+# from google.appengine.ext import ndb
```

Comment:
This commented-out line can also be removed.

Reply:
Done!